plugins/woocommerce-blocks/assets/js/data/checkout/thunks.ts

```diff
@@ -135,3 +139,31 @@ export const __internalEmitAfterProcessingEvents: emitAfterProcessingEventsType
 			}
 		};
 	};
+
+export const updateDraftOrder = ( {
+	orderNotes,
+	additionalFields,
+}: {
+	orderNotes: string;
+	additionalFields: AdditionalValues;
+} ) => {
+	return async ( { registry } ) => {
+		const { receiveCart } = registry.dispatch( CART_STORE_KEY );
+		try {
+			const response = await apiFetchWithHeaders( {
+				path: '/wc/store/v1/checkout',
+				method: 'PUT',
```

Comment:
👍🏻 I'm guessing validation on PUT needs to be more lax to prevent the route blocking partial updates.

Reply:
what do you mean? Validation of additional fields server side?